### PR TITLE
CAKE-5230 | Use new vars

### PR DESCRIPTION
**Pull request by vforge**
## Links

* http://wikia-inc.atlassian.net/browse/CAKE-5230

## Description

- Use new wg variables

## Reviewers

@Wikia/cake

### CODE REVIEW

**Comment by vforge** [comment]
See also https://github.com/Wikia/app/pull/17550

**Comment by vforge** [comment]
Pinging @Wikia/cake 
FYI @Wikia/iwing

File: app/services/affiliate-slots.js [hunk 1]
```diff
@@ -95,6 +96,10 @@ export default Service.extend({
   currentVertical: readOnly('wikiVariables.vertical'),
   currentCountry: readOnly('geo.country'),
 
+  isLaunched: computed('wikiVariables.affiliateUnitEnabledDate', function () {
```
Comment (JoshuaRogan) [inline app/services/affiliate-slots.js:89]: Should this be "private"?

**Review by JoshuaRogan — APPROVED** [review]

File: app/services/affiliate-slots.js (hunk 1 above)
Comment (vforge) [reply app/services/affiliate-slots.js:89]: I was just mirroring what we already heave for existing computed properties /shrug

**Review by rybmat — APPROVED** [review]

File: app/components/watch-show.js [hunk 2]
```diff
@@ -51,7 +57,7 @@ export default Component.extend(
       track({
         action: trackActions.impression,
         category: 'article',
-        label: 'watch-show',
+        label: `watch-${this.wikiVariables.watchShowTrackingLabel}`,
```
Comment (mklucsarits) [inline app/components/watch-show.js]: Will it matter that we're not checking if `this.wikiVariables.watchShowTrackingLabel` is defined?

Comment (vforge) [reply app/components/watch-show.js]: 71f799a

**Review by mklucsarits — APPROVED** [review]

**Pull request merged**